src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -1,8 +1,10 @@
 @using Microsoft.AspNetCore.Components.RenderTree
 <div id="test-selector">
   Select test:
-    <select @bind=SelectedComponentTypeName>
+    @* The id below is needed for some Ignitor tests *@
```

Comment:
We'd be fine without this comment. There are lots of IDs all over the place throughout E2E tests and we don't comment in each case about what's using it. The assumption is that if there's an ID, something uses it. Nobody could change/remove it without breaking tests anyway.

Reply:
I removed the comment, I made it mostly because it's an Ignitor specific limitation (we don't support CSS selectors)